### ISSUE TITLE
Clean-up online documentation generation code

### ISSUE THREAD

**Issue by Lonami** [LonamiWebs/Telethon#1267]
The code that generates https://tl.telethon.dev is a mess and should be cleaned up. In particular, things like "here" being a link are a sign of poor accessibility (the link should be self-descriptive), and "bot index" inside a namespace (e.g. the "here" in https://tl.telethon.dev/methods/account/index.html) lead to a 404.

Generating the bot index should not rely on querying the filesystem and reading every single file to find out the title. The title should be known, and the paths to build the index should also be known even without writing to disk.

The 404's background colour is also different from the main page's background colour, which is distracting.

**Comment by Lonami** [comment]
No concrete plans and does not need to be tracked. If v2 happens it'll naturally be cleaned up.
